examples/docker_submission/docker/model_server.py

```diff
@@ -1,20 +1,22 @@
 from flask import Flask, request, jsonify
+from flask_orjson import OrjsonProvider
 import numpy as np
 import json
 from PIL import Image
 from io import BytesIO
-# import pickle
+import orjson
+
 
 # Import your model here
 from sample_submission import MySimpleScanpathModel
 
 app = Flask("saliency-model-server")
+app.json_provider = OrjsonProvider(app)
```

Comment:
Are we even using the OrjsonProvider? I guess not, since we're manually calling orjson. In this case i would remove the dependency. Alternatively, you can try to keep it, use `jsonify` and set the config option to prevent pretty printing